packages/experimental/composition/package.json

```diff
@@ -35,6 +35,7 @@
   "license": "MIT",
   "dependencies": {
     "@fluentui-react-native/immutable-merge": "^1.1.2",
+    "@fluentui-react-native/use-slot": ">=0.1.1 <1.0.0",
     "@fluentui-react-native/use-slots": ">=0.5.3 <1.0.0",
```

Comment:
Oof, the names of these packages being really similar is unfortunate. What do you think about adding "framework-" in front of "use-slot" to differentiate it a little better?

Reply:
They definitely are. Long term though if we are moving away from compose as the main way to write components, then the use-slots package can go away. Maybe as a second step rename the old one to something a bit more ponderous?

Maybe compose-use-slots or something like that? I'd prefer to do that as a quick PR to not complicate this one but it is great feedback.

Reply:
Sure, that seems fine to me. Out of curiosity are there any issues with renaming existing packages if someone external was using them?

Reply:
If we have users who are using use-slots directly then they would need to rename their references to pick up new versions. In most cases the internals of use-slots have been exposed via re-exporting via the framework package which insulates us from a lot of this.